src/app/clients/form/submitFormHelpers.ts

```diff
@@ -18,107 +22,36 @@ type SubmitFormHelper = (
     primaryKey?: string
 ) => void;
 
-interface PeopleGroupedByAction {
-    [key: string]: Person[];
-}
-
-const personToFamilyRecord = (person: Person, familyID: string): FamilyDatabaseInsertRecord[] => {
-    return Array(person.quantity ?? 1).fill({
+const personToFamilyRecord = (person: Person, familyID: string): FamilyDatabaseInsertRecord => {
+    return {
         family_id: familyID,
         gender: person.gender,
         age: person.age ?? null,
-    });
+    };
 };
 
-const getChildrenInDatabase = async (familyID: string): Promise<string[]> => {
-    const { data, error } = await supabase
-        .from("families")
-        .select("primary_key")
-        .eq("family_id", familyID)
-        .not("age", "is", null);
+const clearFamily = async (familyId: string): Promise<void> => {
+    const { error } = await supabase.from("families").delete().eq("family_id", familyId);
 
     if (error) {
-        const logId = await logErrorReturnLogId("Error with fetch: Children", error);
-        throw new DatabaseError("fetch", "children", logId);
+        const logId = await logErrorReturnLogId(`Error with delete: Family ID ${familyId}`, error);
```

Comment:
(same in other places too please)

```suggestion
        const logId = await logErrorReturnLogId(`Error with delete: Family ID ${familyId}`, { error });
```